packages/fx-core/src/common/spec-parser/specFilter.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+
+import { OpenAPIV3 } from "openapi-types";
+
+const allMethodNames = ["get", "post", "put", "delete", "patch", "head", "options", "trace"];
+const supportedMethodNames = ["get", "post"];
+
+export function specFilter(
+  filter: string[],
+  unResolveSpec: OpenAPIV3.Document
+): OpenAPIV3.Document {
+  const newSpec = { ...unResolveSpec };
+  const newPaths: OpenAPIV3.PathsObject = {};
+  for (const filterItem of filter) {
+    const [method, path] = filterItem.split(" ");
+    const methodName = method.toLowerCase();
+
+    if (!unResolveSpec.paths[path] || !(unResolveSpec.paths[path] as any)[methodName]) {
+      continue;
+    }
+
+    if (!newPaths[path]) {
+      newPaths[path] = { ...unResolveSpec.paths[path] };
+      for (const m of allMethodNames) {
+        delete (newPaths[path] as any)[m];
+      }
+    }
+
+    if (supportedMethodNames.includes(methodName)) {
+      (newPaths[path] as any)[methodName] = (unResolveSpec.paths[path] as any)[methodName];
+
+      // Add the operationId if missing
+      if (!(newPaths[path] as any)[methodName].operationId) {
+        (newPaths[path] as any)[methodName].operationId = `${methodName}${convertPathToCamelCase(
```

Comment:
should here be a whitespace between method name and path?

Reply:
No, it's used for operationId, should not contain whitespace